odp_sdk/client.py

```diff
@@ -11,16 +13,19 @@ class OdpClient(BaseModel):
     token_provider: TokenProvider = Field(default_factory=get_default_token_provider)
 
     _http_client: OdpHttpClient = PrivateAttr()
+    _raw_storage_client: OdpRawStorageClient = PrivateAttr()
+    _resource_client: OdpResourceClient = PrivateAttr()
 
     def __init__(self, **data):
         super().__init__(**data)
 
         self._http_client = OdpHttpClient(base_url=self.base_url, token_provider=self.token_provider)
+        self._raw_storage_client = OdpRawStorageClient(http_client=self._http_client)
+        self._resource_client = OdpResourceClient(http_client=self._http_client)
```

Comment:
You should set the endpoint here, instead of using a default value in the `OdpResourceClient`

Reply:
Fixed